### ISSUE TITLE
[Bug] Timeline isn't loaded anymore after creating 4 events

### ISSUE THREAD

**Issue by widarr** [seanlowe/obsidian-timelines#77]
Hi! I created a few events and tested them in the timeline - they were loaded as normal, but as soon as I added the 4th one nothing is loaded not even the window of the timeline code block...

the events look like this one: 

---
title: Classical Antiquity
description: The age of the first human empires and city states
color: "#A5FF76"
type: background
startDate: 3000-01-01
endDate: 0476-01-01
era: O.E.
path: ""
tags:
  - eras
  - events
  - timeline
showOnTimeline: true
---
#timeline #eras #events

... and the timeline looks like this:

```ob-timeline
tags=timeline;events;eras
divHeight=800
type=flat
```

Tested it also on another computer... When I start Obsidian my events are there in the timeline, but as soon as I click on another note and then again on the one containing the timeline it vanishes again...   Does timelines(revamped) have a problem with events that span over thousands of years?

**Comment by seanlowe** [comment]
hey there, a couple questions:
- where are all the notes located? different files / all same file?
  - all frontmatter notes?
- would it be possible to provide the notes (or at least the events from them) for easy recreation?

Quick debugging check, as well: in the Timelines Revamped plugin settings, all the way at the bottom there's a "Debug Mode". Turn that on. Then push `Ctrl + Shift + I` to open the dev console. try rendering the timeline and see if any red errors show up.

Screenshots if anything looks out of place are helpful.

Hopefully we can get this figured out for you pretty quick

**Comment by widarr** [comment]
I think I have closed in to the error.
Here is a really simple vault where I created 4 background events. the first three worked like normal but as soon as I added the 4th one (Atomic Age) where I changed the date Format from YYYY-MM-DD to YYYY-MM-DDThh:mm:ss it stoped to work...  it seems ob-timeline can't handle mixed format events.
[Dumb_Vault.zip](https://github.com/user-attachments/files/16141054/Dumb_Vault.zip)

**Comment by seanlowe** [comment]
Hi, the dates are a really finicky part of the timeline, especially the horizontal one. The library used to generate the horizontal timeline casts all dates to a real date time object in javascript so I had to run some wonky logic in order to get it to work properly. But that means that it is very strict about how dates have to be formatted.

You can see in the [docs](https://seanlowe.github.io/obsidian-timelines/docs/03_arguments/01_html_arguments/#dates-data-start-date-and-data-end-date) for date arguments, that dates are required to be in the **YYYY-MM-DD-HH** format

**Comment by widarr** [comment]
Just tested it with the YYYY-MM-DD-HH format - this also doesn't work. Maybe this is an upstream problem with the JavaScript vis timeline?

**Comment by seanlowe** [comment]
yeah, the code for this is somewhat atrocious haha. Unfortunately yeah, we are constrained by the upstream. There might be some updates I can make, but I'll have to do some more research and testing on my end. How far down were you wanting to specify?

**Comment by widarr** [comment]
I have ranges of millenia and events down to seconds. There is a program that could do that: https://thetimelineproj.sourceforge.net/  but I don't want to use 2 programs in parallel anymore, it would be nice if I could do everything with Obsidian.

**Comment by weirdwizardthomas** [comment]
Bump up for increasing granularity of timestamps!

**Comment by Flexo013** [comment]
Another bump! Making a timeline of events throughout a single day is currently not really feasible, so minutes (and maybe even seconds) would be awesome to have!

**Comment by rk-kontur** [comment]
YES, please... **BUMP**

**Comment by gleyconJ** [comment]
Please help us make the timeline more precise by including minutes in it, as my timeline needs to be more accurate as I work with days. So within the 24-hour timeline, I also need the minutes.

**Comment by SimonBorin** [comment]
I would like to use this plugin to work on RCA 
For such a proposal, I would ask you to add support not just seconds but milliseconds as well
hh:mm:ss.sss
